### PR TITLE
Downgrade some frequent logs.

### PR DESCRIPTION
**Pull request by chaoticlonghair**
Downgrade some frequent logs.

* Why can **NOT** pass CI, today (2018-04-09)?
  * https://github.com/stepancheg/rust-tls-api/issues/17
  * https://github.com/stepancheg/rust-tls-api/issues/18
  * https://github.com/stepancheg/rust-tls-api/pull/16
  * https://github.com/stepancheg/rust-tls-api/pull/19

### CODE REVIEW

**Review by u2 — COMMENTED** [review]
ci?

File: src/core/cita_bft.rs [hunk 1]
```diff
@@ -884,7 +884,7 @@ impl TenderMint {
                 //proof : self.params vs proposal's block's broof
                 let block_proof = block.get_header().get_proof();
                 let proof = TendermintProof::from(block_proof.clone());
-                info!(" proof is {:?}  {} {}", proof, height, round);
+                debug!(" proof is {:?}  {} {}", proof, height, round);
```
Comment (jerry-yu) [inline src/core/cita_bft.rs:887]: 这个proof打印占用比较多的空间

**Review by jerry-yu — APPROVED** [review]

**Review by driftluo — APPROVED** [review]

**Comment by chaoticlonghair** [comment]
@u2 
Since this repo is just a `git submodule`, we put `Cargo.lock` in [`CITA`](https://github.com/cryptape/cita), so this repo doesn't use `Cargo.lock` to lock versions of crates.
Now, [`tokio-tls-api `](https://github.com/stepancheg/rust-tls-api/tree/master/tokio-tls-api) is broken, so this PR can **NOT** pass CI.

**Comment by TheWaWaR** [comment]
@yangby-cryptape What if put a `Cargo.lock` in current project, will broken cita workspace?

**Review by TheWaWaR — APPROVED** [review]
OK to me

**Comment by u2** [comment]
@yangby-cryptape what about updating cargo.toml to lock the version?

**Pull request merged**